app/controllers/userList.js

```diff
@@ -0,0 +1,16 @@
+const _user = require('./../models').user;
+
+exports.userList = (req, res) => {
+  const usermail = req.body.email;
+  if (_user.findAll({ attributes: ['sesion'] }, { where: { email: usermail } })) {
```

Comment:
why this if?

Reply:
also, return

Reply:
according to the trello card's criteria : "Se debe estar autenticado para consumir dicho recurso."
  
so the field 'sesion' is a boolean which is set on true right after performing a sign in, in theory, if this field returns true, it means the user who is trying to access the service is active.